docs/src/main/java/GettingStarted.java

```diff
@@ -37,23 +37,26 @@
 import static org.terracotta.angela.client.config.custom.CustomConfigurationContext.customConfigurationContext;
 import static org.terracotta.angela.common.clientconfig.ClientArrayConfig.newClientArrayConfig;
 import static org.terracotta.angela.common.distribution.Distribution.distribution;
+import static org.terracotta.angela.common.dynamic_cluster.Stripe.stripe;
+import static org.terracotta.angela.common.provider.DynamicConfigManager.dynamicCluster;
 import static org.terracotta.angela.common.tcconfig.TcConfig.tcConfig;
+import static org.terracotta.angela.common.tcconfig.TerracottaServer.server;
+import static org.terracotta.angela.common.topology.LicenseType.TERRACOTTA_OS;
+import static org.terracotta.angela.common.topology.PackageType.KIT;
 import static org.terracotta.angela.common.topology.Version.version;
 
 /**
  * @author Aurelien Broszniowski
  */
 public class GettingStarted {
-  private static String EHCACHE_OS_VERSION = "3.8.1";
-
   @Test
   public void configureCluster() throws Exception {
     // tag::configureCluster[]
     ConfigurationContext configContext = customConfigurationContext() // <1>
         .tsa(tsa -> tsa // <2>
             .topology(new Topology( // <3>
-                distribution(version(EHCACHE_OS_VERSION), PackageType.KIT, LicenseType.TERRACOTTA_OS), // <4>
-                tcConfig(version(EHCACHE_OS_VERSION), getClass().getResource("/tc-config-a.xml")))) // <5>
+                distribution(version("3.8.1"), PackageType.KIT, LicenseType.TERRACOTTA_OS), // <4>
+                tcConfig(version("3.8.1"), getClass().getResource("/tc-config-a.xml")))) // <5>
```

Comment:
why replacing this? I think that the reason to use EHCACHE_OS_VERSION was not to have to update test for every terracotta release

Reply:
Dynamic config changes are available in certain kits only (e.g. ehcache 3.9-SNAPSHOT, tc-ee 10.7.0-SNAPSHOT). Using these kits automatically translates to `Distribution107Controller`. Using a global `EHCACHE_OS_VERSION` didn't seem correct, because the added tests need to use a different ehcache version

Reply:
you can update the EHCACHE_OS_VERSION field to use 3.9-SNAPSHOT

Reply:
The old angela API cannot be used for dynamic-config kits. The old APIs need to use the non-dynamic-config kits.

Reply:
I'm wondering  what would be the best:
have 2 different tests files for documentation, one for <3.9, the other for >=3.9
or delete the <3.9 test in the documentation
or maybe keep one java file and have two variables EHCACHE_OS_VERSION_OLDER_LINE and EHCACHE_OS_VERSION

but the fact that yo can't use one variable now and needs to hard code the 3.9 version shows that something is missing

Reply:
Introduced a new variable called `EHCACHE_VERSION`. Ambiguous on purpose (didn't like OLDER/NEWER distinction)